make/proto.mk

```diff
@@ -27,41 +27,41 @@ DOCKER_BUF := docker run -v $(shell pwd):/workspace --workdir /workspace bufbuil
+	@$(DOCKER_BUF) breaking --against https://github.com/cheqd/cheqd-node.git#branch=main
 
-# GOGO_PROTO_URL           = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
-# GOOGLE_PROTO_URL         = https://raw.githubusercontent.com/googleapis/googleapis/master
-# COSMOS_PROTO_URL         = https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.4/proto/cosmos
-# COSMOS_ORM_PROTO_URL     = https://raw.githubusercontent.com/cosmos/cosmos-sdk/orm/v1.0.0-alpha.10/proto/cosmos
+GOGO_PROTO_URL           = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
+GOOGLE_PROTO_URL         = https://raw.githubusercontent.com/googleapis/googleapis/master
+COSMOS_PROTO_URL         = https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.4/proto/cosmos
+COSMOS_ORM_PROTO_URL     = https://raw.githubusercontent.com/cosmos/cosmos-sdk/orm/v1.0.0-alpha.10/proto/cosmos
 
-# GOGO_PROTO_TYPES         = third_party/proto/gogoproto
-# GOOGLE_PROTO_TYPES       = third_party/proto/google
-# COSMOS_PROTO_TYPES       = third_party/proto/cosmos
+GOGO_PROTO_TYPES         = third_party/proto/gogoproto
+GOOGLE_PROTO_TYPES       = third_party/proto/google
+COSMOS_PROTO_TYPES       = third_party/proto/cosmos
 
-# proto-swagger-deps:
-# 	@mkdir -p $(GOGO_PROTO_TYPES)
-# 	@curl -sSL $(GOGO_PROTO_URL)/gogoproto/gogo.proto > $(GOGO_PROTO_TYPES)/gogo.proto
+proto-swagger-deps:
+	@mkdir -p $(GOGO_PROTO_TYPES)
+	@curl -sSL $(GOGO_PROTO_URL)/gogoproto/gogo.proto > $(GOGO_PROTO_TYPES)/gogo.proto
 
-# 	@mkdir -p $(GOOGLE_PROTO_TYPES)/api/
-# 	@curl -sSL $(GOOGLE_PROTO_URL)/google/api/annotations.proto > $(GOOGLE_PROTO_TYPES)/api/annotations.proto
-# 	@curl -sSL $(GOOGLE_PROTO_URL)/google/api/http.proto > $(GOOGLE_PROTO_TYPES)/api/http.proto
+	@mkdir -p $(GOOGLE_PROTO_TYPES)/api/
+	@curl -sSL $(GOOGLE_PROTO_URL)/google/api/annotations.proto > $(GOOGLE_PROTO_TYPES)/api/annotations.proto
+	@curl -sSL $(GOOGLE_PROTO_URL)/google/api/http.proto > $(GOOGLE_PROTO_TYPES)/api/http.proto
 
-# 	@mkdir -p $(COSMOS_PROTO_TYPES)/base/v1beta1/
-# 	@curl -sSL $(COSMOS_PROTO_URL)/base/v1beta1/coin.proto > $(COSMOS_PROTO_TYPES)/base/v1beta1/coin.proto
+	@mkdir -p $(COSMOS_PROTO_TYPES)/base/v1beta1/
+	@curl -sSL $(COSMOS_PROTO_URL)/base/v1beta1/coin.proto > $(COSMOS_PROTO_TYPES)/base/v1beta1/coin.proto
 
-# 	@mkdir -p $(COSMOS_PROTO_TYPES)/base/query/v1beta1/
-# 	@curl -sSL $(COSMOS_PROTO_URL)/base/query/v1beta1/pagination.proto > $(COSMOS_PROTO_TYPES)/base/query/v1beta1/pagination.proto
+	@mkdir -p $(COSMOS_PROTO_TYPES)/base/query/v1beta1/
+	@curl -sSL $(COSMOS_PROTO_URL)/base/query/v1beta1/pagination.proto > $(COSMOS_PROTO_TYPES)/base/query/v1beta1/pagination.proto
 
-# 	@mkdir -p $(COSMOS_PROTO_TYPES)/orm/v1alpha1/
-# 	@curl -sSL $(COSMOS_ORM_PROTO_URL)/orm/v1alpha1/orm.proto > $(COSMOS_PROTO_TYPES)/orm/v1alpha1/orm.proto
+	@mkdir -p $(COSMOS_PROTO_TYPES)/orm/v1alpha1/
+	@curl -sSL $(COSMOS_ORM_PROTO_URL)/orm/v1alpha1/orm.proto > $(COSMOS_PROTO_TYPES)/orm/v1alpha1/orm.proto
```

Comment:
Correct me if I'm wrong but this will generate for base Cosmos types and we don't care about that.

Reply:
yep, no need for that as well.